### PR TITLE
Add support for Variant in .buildstamp

### PR DESCRIPTION
**Pull request by poncovka**
Added support for the Variant item in the .buildstamp file, so we can
choose install classes based on the name of the variant.

Give the modules with install classes proper python names, otherwise
it is not possible to import them.

### CODE REVIEW

File: pyanaconda/installclasses/fedora_server.py [hunk 1]
```diff
@@ -1,5 +1,5 @@
 #
-# fedora-server.py
+# fedora_server.py
```
Comment (jkonecny12) [inline pyanaconda/installclasses/fedora_server.py]: Personally I would like to get rid of these file names as comments on top of a file. First of all, why we need them in a first place?

Comment (poncovka) [reply pyanaconda/installclasses/fedora_server.py]: Fixed.

**Comment by poncovka** [comment]
@M4rtinK, we don't have to wait for lorax. The variable `productVariant` is set to an empty string by default.

**Comment by jkonecny12** [comment]
Yeah but I think it would be wise to wait. In case some decision will change or this won't be a way how Lorax can implement it.

**Comment by bcl** [comment]
Looks ok to me. Here's the lorax PR - https://github.com/rhinstaller/lorax/pull/302

**Comment by poncovka** [comment]
The lorax PR was merged: https://github.com/rhinstaller/lorax/commit/dc635a97f66aeadd5e353d141c51a20b0024d391

**Pull request merged**